rdbms/src/main/java/org/spine3/server/storage/jdbc/projection/query/SelectTimestampQuery.java

```diff
@@ -32,10 +31,16 @@
 import java.sql.SQLException;
 
 import static java.lang.String.format;
-import static org.spine3.server.storage.jdbc.projection.query.Constants.NANOS_COL;
-import static org.spine3.server.storage.jdbc.projection.query.Constants.SECONDS_COL;
+import static org.spine3.server.storage.jdbc.projection.query.ProjectionTable.NANOS_COL;
+import static org.spine3.server.storage.jdbc.projection.query.ProjectionTable.SECONDS_COL;
 import static org.spine3.validate.Validate.isDefault;
 
+/**
+ * Query that selects all {@link Timestamp} from the {@link ProjectionTable}.
```

Comment:
Stop, there is only one timestamp in this table. It selects the time of the last handled event.